### ISSUE TITLE
The chart content in 'Food and country' Bar chart is not Keyboard accessible. 

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#19721]
#### Expected behaviour
The chart content in 'Food and country' Bar chart should be Keyboard accessible. And Screen reader focus should move on graph content and announce the graph data

#### Actual behaviour
The chart content in 'Food and country' Bar chart  is not Keyboard accessible. 

With Mouse hover there are tooltips displayed over each 'Graph' data series. But with keyboard user is not able to navigate. 

Screen reader behavior: Neither in focus nor in browse mode focus moves to graph content. 

#### Live demo with steps to reproduce
[Bar chart | nivo](https://nivo.rocks/bar/)

1. Open URL-[Bar chart | nivo](https://nivo.rocks/bar/)
2. Navigate to graphs with keyboard and observe whether the chart content in 'Food and country' Bar chart is Keyboard accessible or not. 

#### Affected browser(s)
OS: Windows 11
OS version: 22H2 (OS Build 22621.2134)
Edge Version: 116.0.1938.69 (Official build) (64-bit)

**Attachments:**
![The chart content in 'Food and country' Bar chart is not Keyboard accessible  ](https://github.com/highcharts/highcharts/assets/93735775/74eb7bc1-5893-4a90-8495-a64329a73daa)

**Comment by karolkolodziej** [comment]
Hi @msftedad! 

I think you've reported the issue under the wrong repository.
We won't be able to help you with the Nivo charts. Please reach out to their creators.
